openaerostruct/utils/plot_wing.py

```diff
@@ -12,36 +12,28 @@
 
 
 import sys
-
-major_python_version = sys.version_info[0]
-
-if major_python_version == 2:
-    import tkFont
-    import Tkinter as Tk
-else:
-    import tkinter as Tk
-    from tkinter import font as tkFont
-
 import numpy as np
 from openmdao.recorders.sqlite_reader import SqliteCaseReader
 
 import matplotlib
+from matplotlib.backends.backend_tkagg import FigureCanvasTkAgg, NavigationToolbar2Tk
+import matplotlib.pyplot as plt
+from matplotlib import cm
+import matplotlib.animation as manimation
 
 matplotlib.use("TkAgg")
 matplotlib.rcParams["lines.linewidth"] = 2
 matplotlib.rcParams["axes.edgecolor"] = "gray"
 matplotlib.rcParams["axes.linewidth"] = 0.5
-from matplotlib.backends.backend_tkagg import FigureCanvasTkAgg, NavigationToolbar2Tk
-import matplotlib.pyplot as plt
-from mpl_toolkits.mplot3d import Axes3D
-from matplotlib import cm
-import matplotlib.animation as manimation
 
-from traceback import print_exc
+major_python_version = sys.version_info[0]
 
-#####################
-# User-set parameters
-#####################
+if major_python_version == 2:

```

Comment:
I don't think we support py2 anymore right?

Reply:
That's right